packages/component/package.json

```
@@ -65,7 +65,7 @@
     "strip-markdown": "^3.0.4"
   },
   "peerDependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react": "16.8.6",
+    "react-dom": "16.8.6"
```

```suggestion
    "react-dom": "^16.8.6"
```

(Don't blame me the suggestions not working for multiline 🙈)